src/app/modules/ix-date/ix-date.component.ts

```diff
@@ -0,0 +1,30 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { utcToZonedTime, zonedTimeToUtc } from 'date-fns-tz';
+import { LocaleService } from 'app/services/locale.service';
+
+@Component({
+  selector: 'ix-date',
+  templateUrl: './ix-date.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class IxDateComponent {
+  /** Must be in browser timezone */
```

Comment:
Let's update comment to `Date must be in browser timezone`, because a number (timestamp) is always UTC.